pkg/reader/rmn_home.go

```diff
@@ -69,12 +75,80 @@ type rmnHomePoller struct {
 	pollingDuration      time.Duration // How frequently the poller fetches the chain configs
 }
 
-func NewRMNHomePoller(
+// GetRMNHomePoller returns a rmnHomePoller instance if it already exists, else creates a new one.
+// Returned rmnHomePoller is always started, therefore user of that API doesn't need to care about
+// concurrency and accidentally starting the same poller multiple times.
```

Comment:
This is a small nit but I wonder if we can return an interface that doesn't have `Start`?

Reply:
Also maybe no `Close` either